src/mcp_server_opendal/__init__.py

```diff
@@ -1,12 +1,12 @@
-from . import server
-import asyncio
 import logging
 
+from . import server
+
 
 def main():
     """Main entry point for the package."""
     logging.basicConfig(level=logging.DEBUG)
-    asyncio.run(server.main())
+    server.main()
```

Comment:
The `main()` function is actually a sync function and don't need to by run by `asyncio`